docs/reference-guides/data/data-core.md

```diff
@@ -876,6 +876,10 @@ _Returns_
 
 -   `Object`: Action object.
 
+### receiveUserPermissions
```

Comment:
Just like `receiveUserPermission`, should this be ignored from documentation?

I'll add some inline docs and `@ignore` it.